splash/benchmark/file_server.py

```diff
@@ -0,0 +1,31 @@
+import SimpleHTTPServer
```

Comment:
It could be better to use Twisted (or some other async framework) because SimpleHTTPServer is single-threaded, and Splash can download multiple resources in parallel. I wonder if it is important for tests, maybe not. 

With Twisted we can also simulate conditions like non-responding servers, delayed responses, etc.